pkg/api/releases.go

```diff
@@ -0,0 +1,75 @@
+package api
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"sync"
+
+	"github.com/google/go-github/v39/github"
+	"golang.org/x/oauth2"
+)
+
+var client *github.Client
+var clientMut sync.Mutex
+
+func InitClient(token string) {
+	clientMut.Lock()
```

Comment:
This is the only place the mutex is used and it's never unlocked.

Reply:
welp

Reply:
and yet it works despite that 😆